src/cli/remove.rs

```diff
@@ -41,21 +42,27 @@ pub async fn execute(args: Args) -> miette::Result<()> {
     match dependency_type {
         DependencyType::PypiDependency => {
             for name in dependency_config.pypi_deps(&project)?.keys() {
-                project.manifest.remove_pypi_dependency(
-                    name,
-                    &dependency_config.platform,
-                    &dependency_config.feature_name(),
-                )?;
+                project
+                    .manifest
+                    .remove_pypi_dependency(
+                        name,
+                        &dependency_config.platform,
+                        &dependency_config.feature_name(),
+                    )
+                    .context("failed to remove pypi dependency")?;
```

Comment:
maybe?

`.context(format!("failed to remove pypi dependency {name}"))`

Reply:
Good Idea!